libpv/EvidenceEnumerator.py

```diff
@@ -180,6 +180,9 @@ def _GetAlias(self, scan_node):
             else:
                 alias = os.path.basename(alias)
 
+        #Remove python3 b'' byte-encoded str wrapper without breaking unicode support
+        if alias.startswith("b'"):
```

Comment:
I think the better solution will be to just encode the binary values to utf. The UI can handle unicode. The reason it is binary is because that is how pytsk holds these values. Better to just encode at the source of the attribute retrieval.

Reply:
When you say at the source of the retrieval, do you mean overriding the name similar to what you did with OSFile.Open function on the test kitchen? 

Also just to clarify which is the preferred representation. Decoded or encoded? Top image is .encode('utf-8') in the property window. Bottom image has nothing done to it which is what it would look like after a .decode()

![2020-04-12-175438_264x65_scrot](https://user-images.githubusercontent.com/8632295/79080955-0fa54380-7ce7-11ea-998f-1a9530092ddd.png)
![2020-04-12-175443_207x35_scrot](https://user-images.githubusercontent.com/8632295/79080956-0fa54380-7ce7-11ea-806e-929f40a19c02.png)

Reply:
So pytsk stores most of its "strings" as bytes instead of str.

For example, here: https://github.com/forensicmatt/PancakeViewer/blob/6b6e9081762857846674997f63e358b7dc189233/libpv/EvidenceEnumerator.py#L141

We see `alias = u'{}'.format(volume.desc)`. `volume` being the source of the of the attribute retrieval. `volume` is a TSK object and `desc` is stored as bytes. That is were we would want to do the decoding. Thus, it should be: `alias = volume.desc.decode("utf8")`.

By doing the decoding at the attribute retrieval source, we still get:
![image](https://user-images.githubusercontent.com/8909830/79081529-850b1700-7cdb-11ea-8136-b6280af61caa.png)

Reply:
Though, to be honest, I would have thought `alias = u'{}'.format(volume.desc)` would have done this. I was under the impression when you use format, it casts to string.

Reply:
Cool, making this change now in the two places this needs to happen.

The problem is str(b'test') is a string type but it preserves the b' ' as part of the string